CHANGELOG.md

```diff
@@ -14,10 +14,22 @@ The types of changes are:
 * `Fixed` for any bug fixes.
 * `Security` in case of vulnerabilities.
 
-## [Unreleased](https://github.com/ethyca/fides/compare/2.6.1...main)
+## [Unreleased](https://github.com/ethyca/fides/compare/2.6.2...main)
+
+
+## [2.6.2](https://github.com/ethyca/fides/compare/2.6.1...2.6.2)
+
+### Fixed
+
+* Issue addressing missing field in dataset migration [#2509](https://github.com/ethyca/fides/pull/2510)
 
 ## [2.6.1](https://github.com/ethyca/fides/compare/2.6.0...2.6.1)
 
+### Fixed
+
+* Fix errors when privacy requests execute concurrently without workers [#2489](https://github.com/ethyca/fides/pull/2489)
+* Enable saas request overrides to run in worker runtime [#2489](https://github.com/ethyca/fides/pull/2489)
+
 ## [2.6.0](https://github.com/ethyca/fides/compare/2.5.1...2.6.0)
```

Comment:
Should you remove from the corresponding sections below?

Reply:
Good spot thanks